libcontainer/devices/devices_unix.go

```diff
@@ -94,6 +94,9 @@ func getDevices(path string) ([]*configs.Device, error) {
 			if err == ErrNotADevice {
 				continue
 			}
+			if os.IsNotExist(err) {
```

Comment:
I'll make a case for my patch. A lower level function can return an err as it sees it but a higher level function can translate/ignore it as it sees fit. `DeviceFromPath` could be used in other contexts where it might be useful to know what exact err it actually saw.